threedi_schema/migrations/versions/0224_upgrade_db_structure_control.py

```diff
@@ -166,19 +166,29 @@ def set_geom_for_control_measure_map():
     for control in ['memory', 'table']:
         control_table = f'{control}_control'
         query = f"""
-        UPDATE 
+        UPDATE
             control_measure_map
-        SET 
+        SET
             geom = (
                 SELECT 
                     MakeLine(tc.geom, cml.geom)
                 FROM 
-                    {control_table} AS tc,
-                    control_measure_location AS cml
```

Comment:
Does the rest of the query still work now that you no longer have defined cml? It is still referred to.

Reply:
Yes, works